setup.py

```diff
@@ -403,31 +498,46 @@ def setup_package():
                 "src/nrnpython",
                 "src/nrnmpi",
             ],
-            extra_compile_args=["-std=c++17"],
-            extra_link_args=[
+            extra_compile_args=extra_compile_args + ["-std=c++17"],
+            extra_link_args=extra_link_args
+            + [
                 # use relative rpath to .data/lib
                 "-Wl,-rpath,{}".format(REL_RPATH + "/.data/lib/")
+            ]
+            if not just_extensions
```

Comment:
This reads pretty bad, please at least drop the negative here, the inline if could also be moved to _inside_ the `format` call and simplify this a wee bit more.